codegen/gateway.go

```diff
@@ -115,12 +117,38 @@ type ClientClassConfig struct {
 	Dependencies Dependencies           `json:"dependencies"`
 }
 
+// helper struct to pull out the fixture config
+type clientClassConfigFixture struct {
+	Config struct {
+		Fixture *Fixture `json:"fixture"`
+	} `json:"config"`
+}
+
 // Dependencies lists all dependencies of a module
 type Dependencies struct {
 	Client []string `json:"client"`
 	//	Service []string `json:"service"`  // example extension
 }
 
+// Fixture specifies client fixture import path and all scenarios
+type Fixture struct {
+	ImportPath string              `json:"importPath"`
```

Comment:
`// ImportPath is the package where the user-defined Fixture global variable is contained. The Fixture object defines, for a given client, the standardized list of fixture scenarios for that client.`

Reply:
done